### ISSUE TITLE
Accessibility Issue: WCAG 2.0 AA Colour Contrast Issue /exercises-oss-hugo.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#741]
**Description**
Highlighted text element does not have sufficient contrast from background

relates to #714 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/exercises-oss-hugo.html
*CSS Selector(s) (if possible and applicable)*:  code
*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

<img width="460" alt="Screen Shot 2020-01-31 at 11 30 07 AM" src="https://user-images.githubusercontent.com/26819992/73556250-0d885400-441d-11ea-95b1-5a0258dac926.png">

**Additional information or thoughts**

As a result of scan #680
